inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/kafka/KafkaSource.java

```diff
@@ -90,6 +90,8 @@ public class KafkaSource extends StreamSource {
 
     @ApiModelProperty(value = "Data field escape symbol")
     private String dataEscapeChar;
+    @ApiModelProperty("Whether wrap content with InlongMessage")
+    private boolean wrapWithInlongMessage = true;
```

Comment:
1. Add one blank line before the new field.
2. Suggest changing the field name to `wrapWithInlongMsg`, because `InlongMsg` is the same as the `InLongMsgXxx`.

```suggestion
    @ApiModelProperty("Whether wrap content with InlongMsg")
    private boolean wrapWithInlongMsg = true;
```